packages/firebase_admob/example/android/app/src/main/java/io/flutter/plugins/firebaseadmobexample/MainActivity.java

```diff
@@ -5,13 +5,23 @@
 package io.flutter.plugins.firebaseadmobexample;
 
 import android.os.Bundle;
+import android.widget.TextView;
+import com.google.android.gms.ads.formats.UnifiedNativeAd;
 import io.flutter.app.FlutterActivity;
+import io.flutter.plugins.firebaseadmob.FirebaseAdMobPlugin;
 import io.flutter.plugins.GeneratedPluginRegistrant;
 
 public class MainActivity extends FlutterActivity {
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     GeneratedPluginRegistrant.registerWith(this);
+    FirebaseAdMobPlugin.setNativeAdGenerator((UnifiedNativeAd ad) -> {
+      final TextView myAdView = new TextView(this);
+      myAdView.setWidth(200);
+      myAdView.setHeight(200);
+      myAdView.setText("This is an ad!");
```

Comment:
Let's show some real add content here

Reply:
I displayed the body of the ad. https://github.com/FirebaseExtended/flutterfire/pull/169/files#diff-729cfe12b0f986dfdcd85db6a241500aR24

![Screenshot_20190923-133427](https://user-images.githubusercontent.com/10687576/65461734-3febca00-de09-11e9-9acd-d61f9cbc8d7a.png)

Reply:
Nvm. I missed a key point on the website that states it's required.